lib/rake/circle.rake

```diff
@@ -25,6 +25,9 @@ RUN_APPS_TESTS_TAG = 'test apps'.freeze
 # Don't run any UI or Eyes tests.
 SKIP_UI_TESTS_TAG = 'skip ui'.freeze
```

Comment:
Is this used anywhere else? If we're making this change long-term, I'd delete this if it's unused.

Reply:
I've commented it out, since I suspect this change is temporary.